### PR TITLE
Update kafka-run-class.bat

### PR DESCRIPTION
**Pull request by jonfreedman**
handle existing classpath with spaces
list each lib dependency individually

### CODE REVIEW

**Comment by jonfreedman** [comment]
I have some IBM\WebSphere MQ jars in the CLASSPATH variable which results in the following error when using the .bat file in the latest release

```\IBM\WebSphere was unexpected at this time.```

File: bin/windows/kafka-run-class.bat [hunk 1]
```diff
@@ -184,7 +186,7 @@ rem echo.
 goto :eof
 :concat
 IF ["%CLASSPATH%"] EQU [""] (
-  set CLASSPATH="%1"
+  set "CLASSPATH=%1"
```
Comment (guozhangwang) [inline bin/windows/kafka-run-class.bat:189]: Just for my own benefits: why do we need to change this?

**Comment by jonfreedman** [comment]
That one is less important, I think only relevant if the user has decided to unpack kafka into a path containing spaces, I'd need to check it - may be completely unnecessary 

---- On Thu, 26 Jan 2017 01:59:32 +0000 notifications@github.com wrote ----

@guozhangwang commented on this pull request.

In bin/windows/kafka-run-class.bat:

> @@ -184,7 +186,7 @@ rem echo.
 goto :eof
 :concat
 IF ["%CLASSPATH%"] EQU [""] (
-  set CLASSPATH="%1"
+  set "CLASSPATH=%1"
Just for my own benefits: why do we need to change this?

**Comment by jonfreedman** [comment]
Running the following hopefully a good example:
```
set "x=foo bar"
set CLASSPATH="%x%"
set "CLASSPATH=%CLASSPATH%;%x%"
echo %CLASSPATH%

set "CLASSPATH=%x%"
set "CLASSPATH=%CLASSPATH%;%x%"
echo %CLASSPATH%
```

It outputs
```
"foo bar";foo bar
foo bar;foo bar
```

**Comment by guozhangwang** [comment]
Merged to trunk and 0.10.2.

**Pull request closed without merging**